mongo_orchestration/sharded_clusters.py

```diff
@@ -56,8 +56,24 @@ def __init__(self, params):
         if self.sslParams:
             self.kwargs.update(DEFAULT_SSL_OPTIONS)
 
+        # Determine what to do with config servers via mongos version.
+        mongos = Server(name='mongos', procParams={})
+        mongos_version = mongos.version
+        mongos.cleanup()
         configsvr_configs = params.get('configsvrs', [{}])
-        self.__init_configsvr(configsvr_configs)
+        self.uses_rs_configdb = (mongos_version >= (3, 1, 2) and
+                                 len(configsvr_configs) == 1)
```

Comment:
You can. `configsvrs` either contains multiple server parameter definitions or a single replica set definition (which may contain multiple entries in its `members`).